tests/test_mux.py

```diff
@@ -603,6 +603,25 @@ def multi_nexthop_test(self, dvs, dvs_route, asicdb, appdb, route, neighbors, ma
                             self.check_route_nexthop(dvs_route, asicdb, route, tunnel_nh_id, True)
                 self.del_route(dvs, route)
 
+            # Check route_updates
+            print("Testing route updates")
+            self.add_route(dvs, route, neighbors)
+            print("setting states to active")
+            for port in mux_ports:
+                self.set_mux_state(appdb, port, "active")
+            print("triggering another route update")
+            self.add_route(dvs, route, neighbors)
```

Comment:
for route update, you may want to try with a different set of neighbors. applying the same route with same set of neighbors may skip processing by routeOrch